CHANGES.rst

```diff
@@ -9,6 +9,11 @@ New Tools and Services
 API changes
 -----------
 
+query.py
+^^^^^^^^
+
+- ``_download_file`` now returns the local file path in all cases (formerly, it would return ``None`` if the file was already cached).  Some corner cases where downloads were not properly continued have been fixed [#3232]
```

Comment:
Something went wrong here

Reply:
🙄 yep bad squash.

Reply:
still bad (in wrong version section), but I'm pushing the changes now.

Reply:
ach, sorry, that was just me that time - I didn't realize the existing Infrastructure label was already under a release section